gradefetcher/gradefetcher.py

```diff
@@ -264,20 +283,23 @@ def grade_user(self, data, suffix=""):
                     for result in grader_response.json()["results"]:
                         if "grade" in result:
                             if result["grade"] > 0:
-                                reason = "Assignment {assignment_id}: <b>Passed</b>.".format(
-                                    assignment_id=result["assignment_id"],
-                                )
+                                reason = self.i18n_service.gettext(
+                                    "Assignment {assignment_id}: <b>Passed</b>").format(
+                                        assignment_id=result["assignment_id"],
+                                    )
                                 reasons.append(reason)
-                            elif result["grade"] == 0:
-                                reason = "Assignment {assignment_id}: <b>Failed</b> - {reason} ".format(
+                            elif result["grade"] == 0: 
+                                reason_api_text = self.i18n_service.gettext(result["reason"])
+                                reason = self.i18n_service.gettext("Assignment {assignment_id}: <b>Failed</b> - {reason_api_text}").format(
```

Comment:
@amirtds I wonder if you should join the two `gettext`s here so it's easier to translate... 
like:

`self.i18n_service.gettext("Assignment {assignment_id}: <b>Failed</b> - ".format(.... )) + reason_api_text`